erpnext/telephony/doctype/call_log/call_log.py

```diff
@@ -157,6 +157,8 @@ def link_existing_conversations(doc, state):
 	"""
 	Called from hooks on creation of Contact or Lead to link all the existing conversations.
 	"""
+	if doc.flags.ignore_auto_link_call_log:
```

Comment:
Is this flag intended for custom apps? Cause I don't see it in Frappe or ERPNext